### ISSUE TITLE
Misguide in documentation (return values)

### ISSUE THREAD

**Issue by ceremcem** [pyos/dg#37]
There is a misguide in documentation:

![image](https://user-images.githubusercontent.com/6639874/28493072-3c24e31e-6f18-11e7-8bc0-294e0cb0b576.png)

This function is obviously expected to return `4` (as it's the last line in the function without any `return` statement before) 

```dg
definitely_not_4 = x ->
    x + 2
    4

print <|  definitely_not_4 40  #=> not 4
```

and it obviously returns `4`: 

![image](https://user-images.githubusercontent.com/6639874/28493086-a8ab4b0e-6f18-11e7-8ae9-8384c5d7354d.png)

...in contrast with the documentation.

**Comment by kirbyfan64** [comment]
I think that was supposed to be humor...

**Comment by ceremcem** [comment]
...a little bit too much humor for a documentation. I just got it :rofl:

**Comment by pyos** [comment]
Yeah, the tutorial is supposed to be a bit tongue-in-cheek, as the fake O'Reilly cover might suggest. Kind of hints that the project is not particularly serious, no? (Probably didn't age well, though.)
